src/components/attendee/NavOverlay/ProfileOverlay/AudioSettings/AudioSettings.tsx

```diff
@@ -0,0 +1,22 @@
+import { Select } from "components/attendee/Select";
```

Comment:
Is it possible to move all these `ProfileOverlay` components outside of the `NavOverlay` one?

I know it's how they're connected, but right now it's getting very hierarchical with `NavOverlay/ProfileOverlay/AudioSettings`, it makes it harder to find components by just looking at `src/components/attendee/`

Reply:
moved these things out [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)